### DISCUSSION TITLE
Entry and Label

### DISCUSSION THREAD

**Discussion by ilmiddin-tuychiev** [TomSchimansky/CustomTkinter#1218] [General]
Entry size automatically changes as to size of Label. 
![image](https://user-images.githubusercontent.com/79243649/218260756-6f2e3213-8d24-41e8-bad8-d8ed80c68346.png)
Like above. How can I make entities same width

**Comment by Wolf-SO** [comment]
What [layout manager](https://www.pythonguis.com/tutorials/create-ui-with-tkinter-grid-layout-manager/) are you using? It looks like `grid`.
Please show us the code you are using!

**Reply by ilmiddin-tuychiev** [reply]
Yeh you are right I'm using grid.
```
self.label = customtkinter.CTkLabel(self.tabview.tab("Tab name"), text="Some sentence that is very very long...")
self.label.grid(row=self.ROW_NUMBER, column=0, padx=20, pady=20)
self.ROW_NUMBER+=1
```
Here entry row number equal to ROW_NUMBER.

**Reply by abhaymakes** [reply]
@ilmiddin17 So, you want the entry box to not change it's size based on the label's length?

**Reply by Wolf-SO** [reply]
> Yeh you are right I'm using grid. ![image](https://user-images.githubusercontent.com/79243649/218262525-5924b897-41d3-4539-b97e-bec9f8ff0f23.png) Here entry row number equal to ROW_NUMBER.

@ilmiddin17 If you show code as a screenshot, people have hard time to type the text themselves to apply the fixes. It's always better to show text as text. Just copy paste, the formatting part is rather easy, see help: [Creating and highlighting code blocks](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks "Creating and highlighting code blocks - GitHub Docs").

To make it short: for python code, you write:

    ```python
    # your snippet here
    ```

**Comment by abhaymakes** [comment]
If yes, then you can update the Label's grid code to this : 

```python
self.label.grid(row = self.ROW_NUMBER, column = 0, padx = 20, pady = 20, columnspan = 2)
```

**Reply by ilmiddin-tuychiev** [reply]
Seems it worked, thanks

**Reply by Wolf-SO** [reply]
@ilmiddin17 If it worked, you should [mark *TheFallen-Cat*'s comment as an answer](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer "Participating in a discussion - GitHub Docs").

**Comment by ilmiddin-tuychiev** [comment]
By the way, I need to do some staff like this
![image](https://user-images.githubusercontent.com/79243649/218266364-0a7c568e-eb8b-40b7-8061-574b1e389162.png)
I have some inputs and need to make some calculations.Here is a very long sentence about what to calculate, and the sentence below is the result of the calculations. I'm using Label and Entry. If you have any advice on how to do this I would appreciate it
